apps/vscode/vscode.py

```diff
@@ -169,6 +167,11 @@ class MacUserActions:
     def command_palette():
         actions.key("cmd-shift-p")
 
+    def find(text: str = None):
```

Comment:
This is defining an override for `user.find`, not `edit.find`.

Reply:
Good catch!
I made a follow up here
https://github.com/talonhub/community/pull/1511